bin/triton-bootstrap

```diff
@@ -92,7 +92,11 @@ fi
 #
 # Register this unconfigured Couchbase instance in Consul for discovery by the configuration/bootstrap agent
 #
-curl -f --retry 7 --retry-delay 3 http://consul:8500/v1/agent/service/register -d "$(printf '{"ID":"couchbase-unconfigured-%s","Name":"couchbase-unconfigured","Address":"%s"}' $MYIPPRIVATE $MYIPPRIVATE)"
+curl -f --retry 7 --retry-delay 3 http://consul:8500/v1/agent/service/register -d "$(printf '{"ID":"couchbase-unconfigured-%s","Name":"couchbase-unconfigured","Address":"%s","checks": [{"ttl": "5900s"}]}' $HOSTNAME $MYIPPRIVATE)"
```

Comment:
Register this Couchbase instance as a service that's awaiting configuration. It gets de-registered later, or will fail out after the TTL health check expires.